src/engine/renderer/tr_shade.cpp

```diff
@@ -143,6 +143,7 @@ static void EnableAvailableFeatures()
 	}
 
 	glConfig2.usingMaterialSystem = r_materialSystem.Get() && glConfig2.materialSystemAvailable;
+	glConfig2.usingBindlessTextures = glConfig2.usingMaterialSystem || r_preferBindlessTextures.Get();
```

Comment:
This seems wrong, if `r_preferBindlessTextures` is `true` and the extension is unavailable or disabled, `glConfig2.usingBindlessTextures` would still be true and would lead to errors/crashes.

Reply:
Good catch. Fixed